docs/index.html

```diff
@@ -0,0 +1,63 @@
+
+<!DOCTYPE html>
+<!--
+  ~ Copyright (c) 2016 Goldman Sachs.
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v1.0
+  ~ and Eclipse Distribution License v. 1.0 which accompany this distribution.
+  ~ The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+  ~ and the Eclipse Distribution License is available at
+  ~ http://www.eclipse.org/org/documents/edl-v10.php.
+  -->
```

Comment:
Is this html file based on the demos in the reveal.js repo? It's fine if it is because reveal.js is MIT licensed. We should adjust the header to match.